llvm/include/llvm/CodeGen/SelectionDAGISel.h

```diff
@@ -81,6 +82,13 @@ class SelectionDAGISel {
                             CodeGenOptLevel OL = CodeGenOptLevel::Default);
   virtual ~SelectionDAGISel();
 
+  /// Returns a (possibly null) pointer to the current BatchAAResults.
+  BatchAAResults *CurrentBatchAA() const {
```

Comment:
```suggestion
  BatchAAResults *getBatchAA() const {
```
Shouldn't capitalize method names.